build-docker/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM alpine
+
+# build depndencies
+RUN apk update && apk add \
```

Comment:
This dependency list is making a biased decision on how to build bitcoin core, primarily on not including qt related dependencies to build `bitcoin-qt`